heat-stack/app/components/ui/heat/CaseSummaryComponents/AnalysisHeader.tsx

```diff
@@ -34,27 +32,29 @@ export function AnalysisHeader(props: { usage_data: any }) {
 	// 		3312125.0171753373
 	// 	]])
 
-	const summaryOutputs = props.usage_data?.get('summary_output')
+	// Extract the summary_output from usage_data
+	const summaryOutputs = usage_data?.summary_output;
```

Comment:
Future issues:
1. Remove `?.` optional chaining operator from `user_data`
2. How do we want to handle `?.` optional chaining operators in general? They indicate that something is not as expected, which will have consequences somewhere in the code. Those cases should be handled.

Reply:
We'll handle this using Zod validation, so we can safely remove the `?.` operator. The schema ensures default values. No need for optional chaining since Zod validates the structure and provides defaults where necessary. The validation is in types/index.ts, ensuring consistency.  

This a example:  export const summaryOutputSchema = z.object({
  estimated_balance_point: z.number().optional().default('N/A'),
  other_fuel_usage: z.number().optional().default('N/A'),
  average_indoor_temperature: z.number().optional().default('N/A'),
  difference_between_ti_and_tbp: z.number().optional().default('N/A'),
  design_temperature: z.number().optional().default('N/A'),
  whole_home_heat_loss_rate: z.number().optional().default('N/A'),
  standard_deviation_of_heat_loss_rate: z.number().optional().default('N/A'),
  average_heat_load: z.number().optional().default('N/A'),
  maximum_heat_load: z.number().optional().default('N/A'),
});

Reply:
Let's cover this in https://github.com/codeforboston/home-energy-analysis-tool/issues/258